logisland-components/logisland-processors/logisland-processor-rest/src/main/java/com/hurence/logisland/rest/processor/lookup/AbstractCallRequest.java

```diff
@@ -213,6 +237,62 @@ void modifyRecord(Record record, Record rsp) {
         return Optional.empty();
     }
 
+    ArrayList<Optional<String>> concatBody(Collection<Record> records, ProcessContext context) {
```

Comment:
This method does not seem general to me. I mean it is usefull only for a specific case so I wonder why it is in AbstractCallRequest.  And this method is currently used only in one processor so there is no reason to have this method in AbstractCallRequest (which should really be general)